apps/gateway/lib/gateway/controllers/curse_of_mirra/configuration_controller.ex

```diff
@@ -9,6 +9,24 @@ defmodule Gateway.Controllers.CurseOfMirra.ConfigurationController do
 
   action_fallback Gateway.Controllers.FallbackController
 
+  def get_current_configuration(conn, _params) do
+    version = Configuration.get_current_version()
+    characters = encode_characters(Configuration.list_characters_by_version(version))
+    game_configuration = Jason.encode!(Configuration.get_game_configuration_by_version(version))
+    consumable_items = Jason.encode!(Configuration.list_consumable_items_by_version(version))
+    map_configuration = Jason.encode!(Configuration.list_map_configurations_by_version(version))
```

Comment:
Don't do any `Jason.encode` until the end, this is the likely culprit of the issue you have with nested things

Reply:
Thanks for the heads up! it worked
Done in [Remove not needed encodes for configs](https://github.com/lambdaclass/mirra_backend/pull/806/commits/126259c4b43e683bb9cb54bb54525dbef8e178c8)